gusto/forcing.py

```diff
@@ -120,9 +128,9 @@ def apply(self, scaling, x_in, x_nl, x_out, **kwargs):
         """
         self.scaling.assign(scaling)
         self.x0.assign(x_nl)
-        mu_scaling = kwargs.get("mu_alpha")
-        if mu_scaling is not None:
-            self.mu_scaling.assign(mu_scaling)
+        stage = kwargs.get("stage")
+        if stage is not None:
+            self.stage.assign(stage)
```

Comment:
`stage` is a bad name for this, I think.

Reply:
Yes, Colin wasn't very happy with it either, I thought I'd keep it like this for now and wait for other suggestions here (such as yours of simply doing ```explicit=True```)